python/lsst/dax/apdb/apdbSql.py

```diff
@@ -851,6 +851,10 @@ def _storeDiaObjects(
         insert_id : `ApdbInsertId`
             Insert identifier.
         """
+        if len(objs) == 0:
+            _LOG.info("No objects to write to database.")
```

Comment:
I think Apdb does not normally log at INFO level in regular conditions. Maybe change this to debug?